spec/controllers/stafftools/assignment_repos_controller_spec.rb

```diff
@@ -5,7 +5,9 @@
   let(:user)    { GitHubFactory.create_owner_classroom_org.users.first }
   let(:student) { GitHubFactory.create_classroom_student               }
 
-  let(:assignment) { create(:assignment, creator: user, organization: user.organizations.first, public_repo: false) }
+  let(:assignment) do
+    Assignment.create(title: 'HTML5', creator: user, organization: user.organizations.first, public_repo: false)
+  end
```

Comment:
@tarebyte BTW, I mentioned this in a previous comment. 
https://github.com/education/classroom/pull/541/files#r58305889

> Strange things happened when I was recording this test case. It's like every time I run this test the assignment here is a different one having a different name and I have to record a new VCR each time, so I changed it into a static one. However, I don't know if it's fine.